dpctl/tensor/libtensor/include/kernels/elementwise_functions/sqrt.hpp

```diff
@@ -69,12 +70,17 @@ template <typename argT, typename resT> struct SqrtFunctor
     resT operator()(const argT &in) const
     {
         if constexpr (is_complex<argT>::value) {
-            // #ifdef _WINDOWS
-            //             return csqrt(in);
-            // #else
-            //             return std::sqrt(in);
-            // #endif
-            return csqrt(in);
+#ifdef USE_SYCL_FOR_COMPLEX_TYPES
+            using realT = typename argT::value_type;
+
+            return exprm_ns::sqrt(exprm_ns::complex<realT>(in));
+#else
+#ifdef _WINDOWS
+            return std::sqrt(in);
+#else
+            return std::sqrt(in);
+#endif
```

Comment:
```suggestion
#else
            return std::sqrt(in);
#endif
```
Unless I am misunderstanding something, it seems the ``#ifdef _WINDOWS`` condition is unnecessary here.

It also seems that the custom ``csqrt`` variants are now unused, and it may be possible to remove them.

Reply:
Thanks for catching this @ndgrigorian . It should be `#ifdef _WINDOWS` then use `csqrt` instead of `std::sqrt`. Will push the change.